t/065-tcp-socket-timeout.t

```diff
@@ -994,3 +994,32 @@ close: 1 nil
 
 --- no_error_log
 [error]
+
+
```

Comment:
We use 3 blank lines to separate adjacent test blocks. Maybe you forgot to run [the reindex script](https://github.com/openresty/openresty-devel-utils/blob/master/reindex) to auto-format your new changes in these `.t` files?